pages/index.js

```diff
@@ -1,8 +1,60 @@
-export default function Home() {
+import styled from 'styled-components';
+import Head from 'next/head';
+import fsPromises from 'fs/promises';
+import path from 'path';
+import Bakery from '../components/Bakery';
+
+export async function getStaticProps() {
+  const filePath = path.join(process.cwd(), 'data.json');
+  const jsonData = await fsPromises.readFile(filePath);
+  const bakeryData = JSON.parse(jsonData);
+
+  return {
+    props: bakeryData,
+  };
+}
+
+export default function Home(props) {
+  const bakeries = props.bakeries;
   return (
-    <div>
-      <main></main>
-      <footer></footer>
-    </div>
+    <>
+      <Head>
+        <title>DAILY BREAD</title>
+      </Head>
+
+      <main>
+        <BakeryListContainer>
+          <h2>All Bakeries</h2>
+          <BakeryList>
+            {bakeries.map((bakery) => (
+              <Bakery
+                name={bakery.name}
+                subtitle={bakery.subtitle}
+                priceRange={bakery.priceRange}
+                score={bakery.score}
+                key={bakery.id}
+              />
+            ))}
+          </BakeryList>
+        </BakeryListContainer>
+      </main>
```

Comment:
```suggestion
        <BakeryListContainer>
          <h2>All Bakeries</h2>
          <BakeryList>
            {bakeries.map((bakery) => (
              <Bakery
                name={bakery.name}
                subtitle={bakery.subtitle}
                priceRange={bakery.priceRange}
                score={bakery.score}
                key={bakery.id}
              />
            ))}
          </BakeryList>
        </BakeryListContainer>
```
I am not sure if it will lead to problems to have two mains, as usually the main element belongs in the Layout component. Try to delete it here and see if anything breaks. If it does i will get back to you :)